### PR TITLE
Update automated-backup.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

File: azure-sql/virtual-machines/windows/automated-backup.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ To use Automated Backup v2, review the following prerequisites:
...
 - The SQL Server VM has been registered with the SQL IaaS Agent extension in [full management mode](sql-agent-extension-manually-register-single-vm.md#upgrade-to-full). 
 -  Automated backup relies on the full [SQL Server IaaS Agent Extension](sql-server-iaas-agent-extension-automate-management.md). As such, automated backup is only supported on target databases from the default instance, or a single named instance. If there is no default instance, and multiple named instances, the SQL IaaS extension fails and automated backup will not work. 
+- Automated backup uses [backup compression](https://docs.microsoft.com/sql/database-engine/configure-windows/view-or-configure-the-backup-compression-default-server-configuration-option?view=sql-server-ver16) by default and you cannot disable it.
 
 ## Settings
 The following table describes the options that can be configured for Automated Backup v2. The actual configuration steps vary depending on whether you use the Azure portal or Azure Windows PowerShell commands.
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/automated-backup.md:52]: Same comment as the other doc, I suggest changing the following paragraph to:

The following table describes the options that can be configured for Automated Backup. The actual configuration steps vary depending on whether you use the Azure portal or Azure Windows PowerShell commands. Note that Automated backup uses [backup compression](https://docs.microsoft.com/sql/database-engine/configure-windows/view-or-configure-the-backup-compression-default-server-configuration-option?view=sql-server-ver16) by default and you cannot disable it.

File: azure-sql/virtual-machines/windows/automated-backup.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ To use Automated Backup v2, review the following prerequisites:
...
 - The SQL Server VM has been registered with the SQL IaaS Agent extension in [full management mode](sql-agent-extension-manually-register-single-vm.md#upgrade-to-full). 
 -  Automated backup relies on the full [SQL Server IaaS Agent Extension](sql-server-iaas-agent-extension-automate-management.md). As such, automated backup is only supported on target databases from the default instance, or a single named instance. If there is no default instance, and multiple named instances, the SQL IaaS extension fails and automated backup will not work. 
+- Automated backup uses [backup compression](https://docs.microsoft.com/sql/database-engine/configure-windows/view-or-configure-the-backup-compression-default-server-configuration-option?view=sql-server-ver16) by default and you cannot disable it.
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/automated-backup.md:49]:
```suggestion
```

**Review by bluefooted — CHANGES_REQUESTED** [review]

**Comment by ujpat** [comment]
@MashaMSFT can you help to merge changes please?

**Review by bluefooted — APPROVED** [review]

**Comment by bluefooted** [comment]
#sign-off

**Pull request merged**